lib/ash/changeset/changeset.ex

```diff
@@ -1040,6 +1040,11 @@ defmodule Ash.Changeset do
       type: {:list, {:or, [:atom, :string]}},
       doc:
         "A list of inputs that, if provided, will be ignored if they are not recognized by the action."
+    ],
+    context: [
+      type: :map,
+      doc:
+        "set context, which can be used in any `Ash.Resource.Change`s configured on the action. (in the `context` argument)"
```

Comment:
Better to use doc from `Ash.global_opts`:
```
"Context to set on the query, changeset, or input"
```
Because the context argument is a different thing. (It is overloaded term in Ash.)

Reply:
That's interesting that it mentions `inputs` here, as the global opts are not merged with the `run_action_opts`

create: 
https://github.com/traveltechdeluxe/ash/blob/6acab04b15333945743bbc715d84b4ae43797a7f/lib/ash.ex#L320

vs

action
https://github.com/traveltechdeluxe/ash/blob/6acab04b15333945743bbc715d84b4ae43797a7f/lib/ash.ex#L726

@zachdaniel might also need to merge those in for actions or update the docs?

Reply:
Yep, we should merge those too! Want to do that in a separate PR?